Makefile

```diff
@@ -19,6 +19,7 @@ vendor: ## Install or update project dependencies
 
 broker: $(SOURCES) ## Build the broker
 	go build -i -ldflags="-s -w" ./cmd/broker
+	go build -i -ldflags="-s -w" ./cmd/migration
```

Comment:
this doesn't feel right. The `broker` Makefile target is the ACTUAL broker binary. This target says build the `broker` binary if any of the `$(SOURCES)` have changed.

I would expect the following:

```Makefile
migration: $(SOURCES) ## Build the migration tool
    go build -i -ldflags="-s -w" ./cmd/migration
```